### ISSUE TITLE
Global style imports

### ISSUE THREAD

**Issue by pi0** [nuxt/framework#6536]
### Describe the feature

Nuxt 2 had a nice module for [style resources module](https://github.com/nuxt-community/style-resources-module) made by @manniL. I believe we could either upgrade the module to Nuxt 3 and link from official docs to support both vite and webpack or add it as a built-in and opt-in feature since seems common enough.

Related: @AngeloSchulerPiletti made a PR to docs (https://github.com/nuxt/framework/pull/6490) to add similar usage with vite.

### Additional information

- [ ] Would you be willing to help implement this feature?
- [X] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://v3.nuxtjs.org/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by manniL** [comment]
I totally agree that this should be popular enough for a built-in (but optional) feature! 👍

**Comment by pi0** [comment]
Opt-in seems reasonable. Would you like to work on it as a built-in feature @manniL?
